codespell_lib/_codespell.py

```diff
@@ -1099,7 +1099,13 @@ def flatten_clean_comma_separated_arguments(
 
 def _script_main() -> int:
     """Wrap to main() for setuptools."""
-    return main(*sys.argv[1:])
+    try:
+        return main(*sys.argv[1:])
+    except KeyboardInterrupt:
+        # User has typed CTRL+C
+        sys.stdout.write("\n")
+        sys.stderr.write(f"Cancelling '{sys.argv[0]}'\n")
```

Comment:
Is a message really needed? Most commands I use just exit upon Ctrl+C, without any message.

Reply:
I'm +1 for the message because it is annoying to see superfluous output from ctrl+c

Reply:
@mwtoews Could you can remove the messages and rebase? We'll try to merge despite the Codecov warning.